spec/features/admin/dashboard_spec.rb

```diff
@@ -192,12 +192,12 @@
           NpcModelWeapon.create(npc_model_id: @npc.id, weapon_id: weapon.id)
         end
       end
-      UserNpcModel.create!(npc_model_id: NpcModel.all[0].id, user_id: @user_1.id)
-      NpcModel.all[0].update(is_saved: 1)
-      UserNpcModel.create!(npc_model_id: NpcModel.all[0].id, user_id: @user_1.id)
-      NpcModel.all[0].update(is_saved: 1)
-      UserNpcModel.create!(npc_model_id: NpcModel.all[0].id, user_id: @user_2.id)
-      NpcModel.all[0].update(is_saved: 1)
+      UserNpcModel.create!(npc_model_id: NpcModel.order(:id)[0].id, user_id: @user_1.id)
+      NpcModel.order(:id)[0].update(is_saved: 1)
+      UserNpcModel.create!(npc_model_id: NpcModel.order(:id)[1].id, user_id: @user_1.id)
+      NpcModel.order(:id)[1].update(is_saved: 1)
+      UserNpcModel.create!(npc_model_id: NpcModel.order(:id)[2].id, user_id: @user_2.id)
+      NpcModel.order(:id)[2].update(is_saved: 1)
```

Comment:
Some of these tests would randomly fail, and it's these six lines that appear to be the issue.